data_steward/tools/migrate_cdm52_to_cdm531.py

```
@@ -76,13 +80,16 @@ def get_upgrade_table_query(client, dataset_id, table_id, hpo_id=None):
     try:
```

nit: consider removing the try catch so it can fail loudly in cases we didn't anticipate

---

I think this makes sense since we are filtering down the list to only CDM

---

It looks like this is useful for tables we don't have schemas for, like data_browser tables etc., so keeping this.